elastic/beats
# libbeat/common/transport: fix log message about TLS

## What does this PR do?



This commit fixes the log message issued by the `test output` command.
Our current TLS verification relies on more than the value of
`tlsConfig.InsecureSkipVerify`, so the previous implementation would
log that TLS was disabled when it was not.

This commit fixes it by checking the value of `config.Verification`.

## Why is it important?



It fix a misleading log message

## Checklist




## Author's Checklist



## How to test this PR locally



1. Configure any output with TLS enabled and `ssl.verification_mode` != `strict` (it always worked for `strict`).
2. Run the `test output` command from your beat
3. You should see the log: `server's certificate chain verification is enabled`
## Related issues



~~## Use cases~~



~~## Screenshots~~



~~## Logs~~
